typescript/infra/package.json

```diff
@@ -18,6 +18,8 @@
     "@hyperlane-xyz/sdk": "4.1.0",
     "@hyperlane-xyz/utils": "4.1.0",
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
+    "@safe-global/protocol-kit": "^4.0.2",
+    "@safe-global/safe-core-sdk-types": "^5.0.2",
```

Comment:
is this required? doesn't seem used but unsure if protocol-kit requires its presence

Reply:
Will double check